Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -162,6 +162,9 @@ public class OrnamentsManager: NSObject {
                                                        margins: options.attributionButton.margins)
         constraints.append(contentsOf: attributionButtonConstraints)
 
+        // Update the image of compass
+        _compassView.updateImage(image: options.compass.image)
```

Comment:
Wouldn't we nil compass image here if we have no custom one? 🤔

Reply:
this updateImage function ignores nil images, in that case it would be a no-op. This is the part that I'm not sure about what is the most common/expected way for swift developers. 

Should I nil check here and not invoke updateImage + make updateImage arguments not optional.
or what it does now?

Reply:
If we first set a non-nil image for a compass, then after that we want to reset this by passing a nil compass image, this won't work.
```
// 1. This should use the custom compass image
mapView.ornaments.options.compass.image = anImage
// later
// 2. I would expect this to use default compass image.
mapView.ornaments.options.compass.image = nil
```

Reply:
I'd also expect the compass image would be reverted to default after resetting it to nil. 

On Android resetting the compass image to `null` seems to be ignored as well. [CompassViewPlugin.kt#L70](https://github.com/mapbox/mapbox-maps-android/blob/f8a72f9f5b49b28d4880dba86b2c9603b423b82d/plugin-compass/src/main/java/com/mapbox/maps/plugin/compass/CompassViewPlugin.kt#L70)

Reply:
I think I'm fine with this as long as both platforms are aligned.